openhands/sdk/conversation/impl/remote_conversation.py

```diff
@@ -330,6 +356,8 @@ def __init__(
                 "initial_message": None,
                 "max_iterations": max_iteration_per_run,
                 "stuck_detection": stuck_detection,
+                "working_dir": working_dir,
+                "persistence_dir": persistence_dir,
```

Comment:
@OpenHands, Based on the content of this PR, we should rewrite `openhands/agent_server/conversation_router.py` and the entire config of `agent_server` so we are setting the `working_dir` and `persistence_dir` in the client-side rather than do it at the global config level.